### PR TITLE
[MIG] fetchmail_thread_default: Migration to 12.0

### PR DESCRIPTION
**Pull request by Rad0van**
Straightforward port to 12.0

### CODE REVIEW

**Comment by Rad0van** [comment]
@chienandalu, @sergio-teruel, @Yajo could you have a look please? Thank you.

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "restructuredtext.confPath": ""
+}
```
Comment (sergio-teruel) [inline .vscode/settings.json]: Remove this file

File: fetchmail_thread_default/README.rst [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fsocial-lightgray.png?logo=github
+    :target: https://github.com/OCA/social/tree/11.0/fetchmail_thread_default
+    :alt: OCA/social
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/social-11-0/social-11-0-fetchmail_thread_default
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/205/11.0
```
Comment (sergio-teruel) [inline fetchmail_thread_default/README.rst]: Use oca-gen-addon-readme OCA utility to regerate this file.
https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-12.0#tasks-to-do-in-the-migration

**Review by sergio-teruel — APPROVED** [review]
Code review, little comments

File: fetchmail_thread_default/README.rst (hunk 2 above)
Comment (Rad0van) [reply fetchmail_thread_default/README.rst]: Done.

**Comment by Rad0van** [comment]
> Code review, little comments

@sergio-teruel, after fighting with squashing commits I hope I got it right. Please check.

**Comment by sergio-teruel** [comment]
Ok for me... :+1:

**Comment by Rad0van** [comment]
> Ok for me... 👍

@sergio-teruel thanx. As soon as it gets merged I plan to migrate to 13.0 as that is in fact the version I need ;-)

**Review by yajo — APPROVED** [review]

**Pull request merged**